ddclient.in

```diff
@@ -1283,7 +1283,7 @@ sub main {
             $result     = 0;
 
         } elsif (!scalar(%config)) {
-            warning("no hosts to update.") unless !opt('quiet') || opt('verbose') || !$daemon;
+            warning("no hosts to update.") if !opt('quiet') || opt('verbose');
```

Comment:
This is a little confusing, it seems like it's checking for the case where `--quiet` and `--verbose` are passed together (if just `--quiet` don't print the warning but if `--quiet --verbose` then do print the warning)? I wouldn't expect those arguments to be compatible but I'm not super familiar with how the arguments are used.

```suggestion
            warning("no hosts to update.") if !opt('quiet');
```

Reply:
Done, thanks!